games-arcade/vvvvvv/vvvvvv-2.0~git.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="Change gravity (instead of jumping) to save your crew"
+DESCRIPTION="Notice requested by the author: VVVVVV is a commercial game! The author has \
+given special permission to make this Haiku version available for free. If you \
+enjoy the game, please consider purchasing a copy at thelettervsixtim.es. \
+\
+VVVVVV is a hit 2010 indie video game from Super Hexagon creator Terry Cavanagh. \
+Play as Captain Viridian, whose must rescue their crew after a teleporter mishap \
+caused them to be seperated in Dimension VVVVVV."
+HOMEPAGE="https://thelettervsixtim.es"
+COPYRIGHT="Terry Cavanagh 2010-2020"
+LICENSE="VVVVVV Source Code License v1.0"
+REVISION="1"
+srcGitRev="3697487f47c3c4d5cae01c7a5b7e8c4c2212e3c6"
+SOURCE_URI="https://github.com/TerryCavanagh/VVVVVV/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="02c131ceee90f92c72e45aed36cb431705ba03465d3a6f31c302f2dc5be55c7a"
+SOURCE_DIR="VVVVVV-$srcGitRev"
+ADDITIONAL_FILES="data.zip"
+ARCHITECTURES="x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	vvvvvv = $portVersion
+	app:vvvvvv$secondaryArchSuffix = $portVersion
```

Comment:
On does one doesn't need $secondaryArchSuffix:
```diff
-	vvvvvv = $portVersion
-	app:vvvvvv$secondaryArchSuffix = $portVersion
+	vvvvvv$secondaryArchSuffix  = $portVersion
+	app:VVVVVV = $portVersion
```

Reply:
vvvvvv$secondaryArchSuffix will tell haikuporter we are building a gcc8 package for the gcc2hybrid
app:* doesn't need a $secondaryArchSuffix (only binaries ending up in $binDir (aside from some exceptions))